src/test/java/org/solid/testharness/http/SolidClientTest.java

```diff
@@ -287,8 +288,8 @@ void parseMembersFails() {
     void deleteResource() {
         final String data = PREFIX + "<http://localhost:3000/> ldp:contains <http://localhost:3000/test>.";
         final Client mockClient = mock(Client.class);
-        final HttpResponse<String> mockResponse = mockStringResponse(200, data);
-        final HttpResponse<Void> mockResponseOk = mockVoidResponse(200);
+        final HttpResponse<String> mockResponse = TestUtils.mockStringResponse(200, data);
+        final HttpResponse<Void> mockResponseOk = TestUtils.mockVoidResponse(200);
```

Comment:
I'm a bit uncertain here too, as deletes would often result in a `204`, but again, it could be appropriate for a mockup to return a `200`.